guide/principles/principles.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: guide
+title: Bitcoin design principles
+description: The key principles to follow when designing Bitcoin products.
+nav_order: 2
+has_children: false
+permalink: /guide/principles/principles/
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+The key principles of designing for Bitcoin
+
+-->
+
+# Bitcoin design principles
+
+As a new technology, Bitcoin offers the opportunity of a decentralized open financial system without intermediaries. To make this a reality, we encourage everyone that is working on products to deliberately support the core principles of designing for Bitcoin. Some of these come from the technology itself, and others from the behaviour and ethos of the community. Although every use case and product is different, all applications should strive to follow these principles. Diverging from them should only be done with very good reason.
+
+
+- [Decentralization](#decentralization)
+- [Inclusion](#inclusion)
+- [Transparency](#transparency)
+- [Self-custody](#self-custody)
+- [Security](#security)
+- [Privacy](#privacy)
+- [Interoperability](#interoperability)
+
+---
+
+## Decentralization
+
+Unlike traditional banking systems, the Bitcoin economy does not require new users to seek permission from anyone. Bitcoin has no central point of control. No one person or entity is in charge. Connecting to any node on the network gives you the same rights and responsibilities, which also ensures that there is no single point of failure.
+
+*Build products that are permissive in terms of what users can do with Bitcoin.* <br>
```

Comment:
I like these italic summaries. You think they could go right below the title of each principle as a one-sentence summary?

Reply:
Good idea. Reorganized and for now formatted with block quotes. df4d515